R/Sound_DMR.R

```diff
@@ -575,12 +575,14 @@ run_model <- function(data, i, Output_Frame, formula, model_type,
                       individual_name_z = ''){
   if (model_type == 'binomial') {
     tryCatch({
-      Output_Frame <- run_binomial(data, i, Output_Frame, formula, 'bobyqa')
+      ith_model_summary <- run_binomial(data, i, Output_Frame, formula, 'bobyqa')
 
       # If that model didn't converge, it tried again with a different optimizer, allows ~20% more model convergence
     },error = function(e){tryCatch({ print(paste(i, "No bobyqa Converge, trying Nelder"))
       # Run the model with Nelder_Mead optimizer
-      Output_Frame <- run_binomial(data, i, Output_Frame, formula, 'Nelder_Mead')
+      ith_model_summary <- run_binomial(data, i, Output_Frame, formula, 'Nelder_Mead')
+	  
+	  Output_Frame <- save_model_summary(i, Output_Frame, ith_model_summary,individual_name_z)
```

Comment:
nit: fix indentation

Reply:
Looks fine in mine! weird. I'll just remove and rewrite it..

Reply:
sometimes the tab character \t and spaces ' ' get rendered differently depending on computer settings; some computers may treat tab as two, four, or eight spaces .  Many linters will actually replace tabs with spaces so there's no ambiguity about it.